llvm/labs/CMakeLists.txt

```diff
@@ -1,10 +1,10 @@
-FILE(GLOB children RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_SOURCE_DIR}/*)
```

Comment:
Please revert changes in this file. I suspect that default new line sequence in your patch is windows like \r\n, while unix style \n is required.